x-pack/plugins/cases/server/client/attachments/bulk_get.ts

```diff
@@ -122,12 +122,21 @@ const constructErrors = ({
   const errors: BulkGetAttachmentsResponse['errors'] = [];
 
   for (const soError of soBulkGetErrors) {
-    errors.push({
-      error: soError.error.error,
-      message: soError.error.message,
-      status: soError.error.statusCode,
-      attachmentId: soError.id,
-    });
+    if (isSODecoratedError(soError.error)) {
+      errors.push({
+        error: soError.error.output.payload.error,
+        message: soError.error.output.payload.message,
+        status: soError.error.output.statusCode,
+        attachmentId: soError.id,
+      });
+    } else {
+      errors.push({
+        error: soError.error.error,
+        message: soError.error.message,
+        status: soError.error.statusCode,
+        attachmentId: soError.id,
+      });
+    }
```

Comment:
nit: this duplicated code(also in `x-pack/plugins/cases/server/client/cases/bulk_get.ts`) could probably be an util.